tests/conftest.py

```diff
@@ -39,6 +39,9 @@ def pytest_addoption(parser):
     os.environ['PIP_DISABLE_PIP_VERSION_CHECK'] = '1'  # do not pollute stderr with upgrade advisory
     parser.addoption('--run-integration', action='store_true', help='run the integration tests')
     parser.addoption('--only-integration', action='store_true', help='only run the integration tests')
+    if platform.python_implementation() == 'PyPy' and sys.version_info[0] == 2 and os.name == 'nt':
+        # pytest-xdist does not work on Windows PyPy2, disable it => last compatible: pytest 4 + pytest-xdist 1.31
```

Comment:
Do we require a more modern version of pytest, or can we express this in the test requirements with environment specifiers and/or constraints files?

Reply:
Not that I'm aware of because not installing this package will cause pytest to complain about-n not being a valid argument. IMHO this is the cleanest.

Reply:
I meant actually install `pytest-xdist`, just constrain the versions. The comment makes it sound like if we installed `pytest` version 4 and `pytest-xdist` version 1.31, it would work.

Reply:
The comment is that the patch works for that version combo, pytest-xdist is just not supported for pypy2 windows 🤷‍♂️